packages/wonder-blocks-dropdown/src/components/select-opener.js

```diff
@@ -268,6 +273,11 @@ const _generateStyles = (light, placeholder) => {
                 borderWidth: 1,
                 color: placeholder ? offBlack64 : offBlack,
             },
+            invalid: {
+                background: `${mix(fade(Color.red, 0.06), Color.white)}`,
```

Comment:
@marianmwang this color is wrong. Please use `mix(fade(Color.red, 0.08)` or even better, `fadedRed8` from `tokens.color.`

https://github.com/Khan/wonder-blocks/blob/8abbd5bd7645e8cbe9f18a89b59e13001f66b66e/packages/wonder-blocks-theming/src/tokens.ts#L23